src/service/action-impl.js

```diff
@@ -271,9 +273,17 @@ export class ActionService {
       return;
     }
 
+    // Check if this is an element has a known action.
+    const supportedActions = ELEMENTS_ACTIONS_MAP_[lowerTagName];
+    if (target[ACTION_QUEUE_] && supportedActions &&
```

Comment:
The problem with initializing `ACTION_QUEUE_` here is basically initialize it to what? An empty array? This case specifically will probably be reached if the element uses `installActionHandler` method to initialize the `ACTION_QUEUE_` to be an object with a fake `push` method to allow executing the handler passed. 

So wouldn't initializing it here basically fail silently for these cases? We could throw a better error when not initialized - basically `user().error('Action queue not initialized, probably tried to execute before calling installActionHandler on %s', this.element)` - or something like that. No?